src/core/src/pass/sdpa_to_paged_attention.cpp

```diff
@@ -53,8 +53,28 @@ bool ov::pass::SDPAToPagedAttention::run_on_model(const std::shared_ptr<ov::Mode
 
     auto sliding_window = v0::Constant::create(element::i32, Shape{}, {0});  // sliding_window
 
+    auto has_parameter = [=](const std::shared_ptr<ov::Model>& model,
+                             const std::string& name) -> std::shared_ptr<v0::Parameter> {
+        for (const auto& param : model->inputs()) {
+            const auto& names = param.get_names();
+            if (names.count(name)) {
+                if (auto casted_param = ov::as_type_ptr<v0::Parameter>(param.get_node_shared_ptr())) {
+                    return casted_param;
+                } else {
+                    OPENVINO_THROW("The model is in the inconsistent state. Found input '",
+                                   name,
+                                   "', but couldn't cast it to v0::Parameter.");
+                }
+            }
+        }
+
+        return nullptr;
+    };
+
+    auto input_ids_name = has_parameter(model, "input_ids") ? "input_ids" : "inputs_embeds";
```

Comment:
probably it's better to 
```
auto input_ids_node = get_parameter(model, {possible_name_1, possible_name_2}); // returns the first match
if (!input_ids_node) {
    return false;
}
```

Reply:
I don't really like the idea of passing a set to a function having name `get_parameter`, so I'd rather do it in a different way, which also looks viable to me.

Reply:
```
    std::shared_ptr<v0::Parameter> input_ids_node;
    for (const auto& name : {"input_ids", "inputs_embeds"}) {
        if (input_ids_node = get_parameter(model, name)) {
            break;
        }
    }

    if (!input_ids_node) {
        OPENVINO_THROW("The model doesn't contain input_ids or input_embeds input. Aborting.");
        return false;
    }
```